loads/main.py

```diff
@@ -1,66 +1,20 @@
 import sys
-# import argparse
 from argparse import ArgumentParser
 from loads.fetchconf import fetch
 from loads.loadconf import load_config
-from loads.utils import process 
-
-from pprint import pprint
-
-
-def plan_list():
-    dummy_out = """
-    Pulling CoreOS AMI info...
-    Verifying database setup.
-
-    Finished database setup.
-    Listening on port 8080...
-    Found 0 instances to recover.
-    Finished initializing: None.
-
-    --------------------------
-    PROJECT: Autopush Architecture
-    --------------------------
-
-    PLANS
-    """
-    print(dummy_out)
-    return [
-        '1. Long Connection Test - 80',
-        '2. Long Connection Test - 40 - us east 1',
-        '3. Long Connection Test - 80 - us west 1',
-        '4. Long Connection Test - 80 - us west 2',
-        '5. Long Connection Test - 80 - us west 2 - bigger',
-        '6. Loadtest',
-    ]
+from loads.run import run_test
 
 
 def main(argv=None):
     if argv is None:
         argv = sys.argv
 
-    # it doesn't parse -h and print help.
-    """
-    conf_parser = argparse.ArgumentParser(
-        description=__doc__, # printed with -h/--help
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-        add_help=False
-        )
-    """
     conf_parser = ArgumentParser(prog='loads-cli')
-
-    conf_parser.add_argument("-f", "--fetchconf",
-                             help="Fetch config (json) file from GitHub repo\n"
-                             "i.e.: <repo owner>/<repo name>")
-    conf_parser.add_argument("-l", "--loadconf",
-                             help="Load config (json) file\n"
-                             "use same format ast for fetchconfg\n"
-                             "i.e.: <repo owner>/<repo name>")
     conf_parser.add_argument("-c", "--config-overview",
                              help="Return summary overview of project config")
     conf_parser.add_argument("-r", "--run",
-                             help="Launches interactive testplan runner",
-                             action='store_true')
+                             help="Launches interactive testplan runner"
+                             "i.e.: <repo owner>/<repo name>")
     conf_parser.add_argument("-a", "--abort",
```

Comment:
Additional args haven't been implemented yet, so we should probably comment them out for now (though they will be next on the docket).